### PR TITLE
Add default tags

### PR DESCRIPTION
**Pull request by paurosello**
### Checklist

- [x] Updated CHANGELOG.md.

### Trigger E2E tests

/run cluster-test-suites

### CODE REVIEW

**Comment by paurosello** [comment]
There is an issue with irsa operator with repeated tags https://github.com/giantswarm/irsa-operator/blob/9c4c0aa7b0a95075c32561e44ef94f9554c3afe1/pkg/aws/services/s3/bucket.go#L58

we need to remove installation like https://github.com/giantswarm/irsa-operator/blob/9c4c0aa7b0a95075c32561e44ef94f9554c3afe1/pkg/aws/services/s3/bucket.go#L65

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## Changed
+
+- Add default tags for AWS resources
+
```
Comment (fiunchinho) [inline CHANGELOG.md:13]: Can we be a bit more specific/verbose here?

```suggestion
## Added

- Add the Management Cluster name as a tag to the AWS resources created by CAPA.
- Add the node pool name as a tag to the AWS resources associated with the node pool.

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## Added
```
Comment (fiunchinho) [inline CHANGELOG.md:10]:
```suggestion
### Added
```

**Review by fiunchinho — APPROVED** [review]

**Pull request merged**